app.config.yaml

```diff
@@ -65,3 +66,5 @@ application:
           productScraperRule:
             trigger: "productScraperTrigger"
             action: "fetch-all-products"
+  hooks:
+    post-app-deploy: "node scripts/disable-rules.js"
```

Comment:
I am not sure if this is the best idea. 
We want the rules to be disabled when initially created, but we don't want to disable them on every deployment.

If this is not possible with an app builder manifest, I would rather 

(a) comment them out per default in the manifest so that customers have to explicitly opt-in to create them once the functionality was verified to be working, or 
(b) remove them completely and have the user create them manually.

wdyt?